packages/connectors/react/src/inject.jsx

```diff
@@ -63,7 +63,9 @@ export default injectFunction => WrappedComponent => class extends Component {
   inject = (nextProps) => {
     this.setState(state => ({
       ...state,
-      injectedProps: injectFunction(this.context.store, nextProps || this.props),
+      injectedProps: injectFunction
+        ? injectFunction(this.context.store, nextProps || this.props) || {}
```

Comment:
default object should be a constant (outside the main function) so the shallowEquals is optimized

Reply:
👌